src/pyskindose/plotting/create_dose_map_plot.py

```diff
@@ -1,21 +1,256 @@
+                 for ind in range(len(patient.r))]
+
+    hover_text = [lat_text[cell] + lon_text[cell] + ver_text[cell] +
+                  dose_text[cell] for cell in range(len(patient.r))]
+
+    # create mesh object for the phantom
+    phantom_mesh = [
+        go.Mesh3d(
+            x=patient.r[:, 0], y=patient.r[:, 1], z=patient.r[:, 2],
+            i=patient.ijk[:, 0], j=patient.ijk[:, 1], k=patient.ijk[:, 2],
+            intensity=patient.dose, colorscale=DOSEMAP_COLORSCALE,
+            showscale=True,
+            hoverinfo='text',
+            text=hover_text, name="Human",
+            colorbar=dict(tickfont=dict(color=COLOR_PLOT_TEXT),
+                          title="Skin dose [mGy]",
+                          titlefont=dict(
+                            family=PLOT_FONT_FAMILY,
+                            color=COLOR_PLOT_TEXT)))]
+
+    # Layout settings
+    layout = go.Layout(
+        height=PLOT_HEIGHT,
+        width=PLOT_WIDTH,
+        margin=PLOT_MARGINS,
+
+        font=dict(
+            family=PLOT_FONT_FAMILY,
+            color=COLOR_PLOT_TEXT,
+            size=PLOT_FONT_SIZE),
+
+        hoverlabel=dict(
+            font=dict(
+                family=PLOT_HOVERLABEL_FONT_FAMILY,
+                size=PLOT_HOVERLABEL_FONT_SIZE)),
+
+        titlefont=dict(
+            family=PLOT_FONT_FAMILY,
+            size=PLOT_FONT_SIZE,
+            color=COLOR_PLOT_TEXT),
+
+        paper_bgcolor=COLOR_CANVAS,
+
+        scene=dict(
+            aspectmode=PLOT_ASPECTMODE_PLOT_DOSEMAP,
+            xaxis=dict(
+                title='',
+                backgroundcolor=COLOR_CANVAS,
+                showgrid=False,
+                zeroline=False,
+                showticklabels=False,
+                ),
+
+            yaxis=dict(
+                title='',
+                backgroundcolor=COLOR_CANVAS,
+                showgrid=False,
+                zeroline=False,
+                showticklabels=False,
+                ),
+
+            zaxis=dict(
+                title='',
+                backgroundcolor=COLOR_CANVAS,
+                showgrid=False,
+                zeroline=False,
+                showticklabels=False,
+                ),
+                )
+        )
+
+    # create figure
+    fig = go.Figure(data=phantom_mesh, layout=layout)
+
+    if settings.plot.interactivity:
+        fig.show()
+
+    else:
+
+        if settings.plot.notebook_mode:
+            from tqdm import tqdm_notebook as pbar
+        else:
+            from tqdm import tqdm as pbar
+
+        eyes = [
+            PLOT_EYE_RIGHT, PLOT_EYE_BACK, PLOT_EYE_LEFT, PLOT_EYE_FRONT]
+
+        names = PLOT_ORDER_STATIC
+        file_type_static = PLOT_FILE_TYPE_STATIC
+
+        for i in pbar(range(len(eyes)), desc=f'saving static dosemaps: '):
+            fig['layout']['scene']['camera'] = eyes[i]
+            fig.write_image(f"{names[i]}.png")
+
+        if settings.plot.notebook_mode:
```

Comment:
Perhaps extract the plotting for `notebook_mode` done here into a separate function?

Reply:
Yes! dc2f659f56a27695e3b95966e686a332157ec0af